core/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "core"
+version = "0.1.0"
+authors = ["Alexander Herrmann <alex@gnosis.io>", "Ben Smith <ben@gnosis.io>", "Felix Leupold <felix@gnosis.io>"]
```

Comment:
Maybe now is a good time to come up with the correct "authors" field? Maybe "Gnosis Protocol Developers"?

Reply:
Do we even need authors? We don't publish these crates.

Reply:
I’m ok either way

Reply:
I’m ok either way

Reply:
Seems like a good time to update the authors!